Tasks/DownloadBuildArtifacts/main.ts

```diff
@@ -126,12 +128,28 @@ async function main(): Promise<void> {
                 // Triggering build info not found, or requested, default to specified build info
                 projectId = tl.getInput("project", true);
                 definitionId = definitionIdSpecified;
-                buildId = parseInt(tl.getInput("buildId", true));
+                buildId = parseInt(tl.getInput("buildId", false));
```

Comment:
we should make sure buildId is specified when the option chosen is specific build.